docs/horizen_eon/05-staking/03-how_to_stake_on_eon.md

```diff
@@ -106,7 +106,33 @@ Make a `POST` request to `/block/startForging` to command the node to start forg
 To stop forging, call the endpoint `/block/stopForging`.
 
 #### Stake Maturity
-When a forger stake is created, it is not immediately ready to let the forger take part in the forgers' election. The stake needs **2 consensus epochs** before reaching the maturity period. Afterwards, the forger will be ready to take part in the slot leader lottery and be able to forge new blocks. 
+When a forger stake is created, it is not immediately ready to let the forger take part in the forgers' election. The stake will start to be accounted in the slot leader lottery from the beginning of the  **second consensus epoch** following the one where its creation happened.
+For example: if the stake transaction was included in the middle of epoch 10, it will be accounted from the beginning of the epoch 12.
 
 #### Verify the Stake Creation
 To make sure the operation was successful, make a `POST` request to `/transaction/allForgingStakes` and verify the publicKey used to create the forger stake appears in the returned list.
+
+#### EON fee redistribution system
+The probability that a forger will be elected as a slot leader will be proportional to the amount of stake he owns versus the total stake present on the chain.\
+If a forger wins the forger's lottery and is able to produce a new block, and the block is accepted by the other peers, it will earn a fee reward.\
+The transactions' fee mechanism in EON mimics the one defined in Ethereum. Every transaction has a fee computed with the following formula:
+```
+transaction fee = gasUsed * baseFeePerGas + gasUsed *maxPriorityFeePerGas
+```
+- baseFeePerGas is a minimum rate a user has to pay to include their transaction in the next block. This value is generated with every block header by the protocol and may change block by block, based on how full the previous block was. 
+- The other part of the fee is determined by maxPriorityFeePerGas and represents the so called "forger tip": it is an additional fee the transaction submitter is willing to pay to have its transaction included as soon as possible.
+- there is another user specified parameter in the transaction:
+maxFeePerGas - represents the maximum fee a user is willing to pay for a transaction.
+This parameter actually limits the maxPriorityFeePerGas with tthis rule:\
+if ( baseFeePerGas + maxPriorityFeePerGas) > 	maxFeePerGas, then maxPriorityFeePerGas will be reduced in order to maintain the upper bound of the 	maxFeePerGas.
+- Finally, we can have also legacy (non EIP-1559) transactions, with only a gasPrice parameter that we use for both maxFeePerGas and maxPriorityFeePerGas values
+
```

Comment:
I'm fine in leaving it as is, you could have used a reference to the original EIP to save some time.